streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -446,6 +446,12 @@ void print(final Serde<K> keySerde,
      * @param printed options for printing
      */
     void print(final Printed<K, V> printed);
+    
+    /**
+     * Merges the given streams into one larger stream. 
+     * @param StreamsBuilder instance and streams wanted to be merged.
```

Comment:
Parameter not correct.

Can you also add the sentence about no ordering guarentee (cf. original JavaDoc from `StreamsBuilder`).